cli/services/manager.go

```diff
@@ -34,7 +43,14 @@ func NewServiceManager() ServiceManager {
 }
 
 // AddServicesToCompose adds services to a running docker compose
-func (sm *DockerServiceManager) AddServicesToCompose(profile string, composeNames []string, env map[string]string) error {
+func (sm *DockerServiceManager) AddServicesToCompose(ctx context.Context, profile string, composeNames []string, env map[string]string) error {
+	if enableInstrumentation {
```

Comment:
@axw is there a way to avoid repeating this flag across the project? In other words, is it possible to instrument thee Go app but having the instrumentation code (spans and transactions creation) not doing its job, maybe skipping/or falling back to "dev/null"?

Reply:
You can set the environment variable `ELASTIC_APM_ACTIVE=false`: https://www.elastic.co/guide/en/apm/agent/go/current/configuration.html#config-active

Reply:
Done, thanks!